src/electron/build_action.sh

```diff
@@ -16,7 +16,7 @@
 
 npm run do src/www/build_electron
 
-webpack --config=src/electron/electron_main.webpack.js \
+npx webpack --config=src/electron/electron_main.webpack.js \
```

Comment:
I don't get why this is needed. The bin/ folder is already in the path.

Reply:
Is that a thing you have set up? Because it's not in mine

Reply:
`npm run do` calls the script from npm, and all the dependency binaries are available.

Are you trying to run the scripts directly? That was not really the way it was intended.